docs/UserGuide.adoc

```diff
@@ -14,8 +14,8 @@ Please refer to the <<DeveloperGuide#setting-up, Setting up>> section to learn h
 
 *Using IntelliJ*
 
-. Find the project in the `Project Explorer` (usually located at the left side).
-.. If the `Project Explorer` is not visible, press ALT+1 for Windows/Linux, CMD+1 for macOS to open the `Project Explorer` tab.
+. Find the project in the `Project Explorer` (usually located at the left side)
+.. If the `Project Explorer` is not visible, press ALT+1 for Windows/Linux, CMD+1 for macOS to open the `Project Explorer` tab
```

Comment:
You can use `<kbd>` tag to style keyboard shortcuts? <kbd>&#8984;</kbd>+<kbd>1</kbd> `<kbd>&#8984;</kbd>+<kbd>1</kbd>`

Reply:
Can't seem to use the `<kbd>` tag for `adoc` files, but we can add in the `:experimental:` attribute to do this ([here](https://asciidoctor.org/docs/asciidoc-syntax-quick-reference/#ui-macros))